less/bookmarksToolbar.less

```diff
@@ -44,7 +44,7 @@
     color: black;
     cursor: default;
     font-size: 11px;
-    line-height: 11px;
+    line-height: 14px;
```

Comment:
This looks great when bookmark toolbar icons is turned on but when there is not it throws off the vertical alignment a lot.  Could you fix? Thanks!